CSharp/BatchExplorer/Models/ModelBase.cs

```diff
@@ -106,13 +106,17 @@ private List<PropertyModel> ObjectToPropertyModelRecursive(object o, List<string
                         //Don't track null properties in order to keep the list short
                         //propertyModel = new SimplePropertyModel(propInfo.Name, string.Empty);
                     }
+                    else if ((propertyType == typeof(TimeSpan) || propertyType == typeof(TimeSpan?)) && propertyValue.Equals(TimeSpan.MaxValue))
+                    {
+                        propertyModel = new SimplePropertyModel(propInfo.Name, "Unlimited");
```

Comment:
unlimited explainer:

[Azure/azure-batch-samples] TimeSpan.MaxValue format (#130)
itowlson   4/17/16   Keep this message at the top of your inbox  Groups
To: Azure/azure-batch-samples
notifications@github.com
Introduces special formatting for TimeSpan.MaxValue, so that if fields such as MaxWallClockTime and RetentionTime are unlimited (as is often the case), they are displayed as "Unlimited" rather than "109000 days."

You can view, comment on, or merge this pull request online at:
  https://github.com/Azure/azure-batch-samples/pull/130